src/components/IndiaStateMap/IndiaMapStats.js

```diff
@@ -1,47 +1,59 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import {motion} from 'framer-motion';
+import { motion } from 'framer-motion';
+import * as population from '../../data/Population.json';
 
-const IndiaMapStats = ({ stats, handleAttributeClick, activeAttribute}) => {
-  const {t} = useTranslation();
+const IndiaMapStats = ({ stats, handleAttributeClick, activeAttribute, mode, state }) => {
+  const { t } = useTranslation();
+
+  function calculateCases(cases) {
```

Comment:
If we pass the `mode` as a parameter in this function, we can remove this function all together from the component and then with each render the function would not be redefined and we will get a tiny bit of performance improvement.

Reply:
Done